webui/react/src/omnibar/modifiers/anchor/BaseRenderer.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
```

Comment:
**nit:** `omnibar/modifier/anchor` path seems a bit odd... what about just `src/omnibar/OmnibarRenderer.tsx`?

Reply:
I was keeping same path the package used but we don't need to do that. I like your suggestion, how about we name it ItemRenderer as it only renders a single item/option